src/Core/Binding/BindingConfiguration.cs

```diff
@@ -24,28 +24,31 @@ namespace SonarLint.VisualStudio.Core.Binding
 {
     public sealed class BindingConfiguration : IEquatable<BindingConfiguration>
     {
-        public static readonly BindingConfiguration Standalone = new BindingConfiguration(null, SonarLintMode.Standalone);
+        public static readonly BindingConfiguration Standalone = new BindingConfiguration(null, SonarLintMode.Standalone, null);
 
-        public static BindingConfiguration CreateBoundConfiguration(BoundSonarQubeProject project, SonarLintMode sonarLintMode)
+        public static BindingConfiguration CreateBoundConfiguration(BoundSonarQubeProject project, SonarLintMode sonarLintMode, string BindingConfigDirectory)
         {
             if (project == null)
             {
                 throw new ArgumentNullException(nameof(project));
             }
 
-            return new BindingConfiguration(project, sonarLintMode);
+            return new BindingConfiguration(project, sonarLintMode, BindingConfigDirectory);
```

Comment:
Is it valid for the config path to be null?  Is there a reason for not checking it?

Reply:
I didn't want to change many tests. Fixed.